roles/reproducer/tasks/main.yml

```diff
@@ -176,6 +176,18 @@
           -e @scenarios/reproducers/networking-definition.yml
           playbooks/01-bootstrap.yml
 
+    - name: Install dev tools from install_yamls on controller-0
+      delegate_to: controller-0
+      environment:
+        ANSIBLE_LOG_PATH: "~/ansible-bootstrap.log"
+      no_log: true
+      ansible.builtin.command:
+        chdir: "/home/zuul/src/github.com/openstack-k8s-operators/install_yamls/devsetup"
+        cmd: >-
+          ansible-playbook -i ~/ci-framework-data/artifacts/zuul_inventory.yml
```

Comment:
I don't think install_yamls playbook is build to be able to consume our inventory. Given how simple it is, how we were consuming it and how install_yamls consumes it, why not pass `-i localhost, -c local` instead of our inventory there? Or, if preferred, directly consume the make target (not sure if the make module support the `creates` parameter.